api/lib/notifications.js

```diff
@@ -105,6 +105,10 @@ function getMessageTitle(networks, notification) {
 
     case eventTypes.LUNIE_UPDATE:
       return data.title
+    case eventTypes.SLASH:
+      return `Validator ${data.operatorAddress} got slashed ${data.amount}.`
+    case eventTypes.LIVENESS:
+      return `Validator ${data.operatorAddress} was offline ${data.blocks}.`
```

Comment:
do we have any liveness events in the db?

Reply:
ah on kava. why only kava?

Reply:
only tested on kava